src/99-appendices/09-glossary.md

```diff
@@ -0,0 +1,26 @@
+# Appendix IX: Glossary
+
+A number of terms are used throughout the specification
+
+- **file extension**:
+   a portion of the the file name after the left-most period (`.`) preceded by
+   any other alphanumeric (so `.gitignore` does not have a suffix)
+- **<index>**:
+   a numeric value, possibly prefixed with arbitrary number of 0s for consistent
+   indentation, e.g. it is `01` in `run-01` following `run-<index>` specification
+- **<label>**:
+   an alphanumeric value, possibly prefixed with arbitrary number of 0s for consistent
+   indentation, e.g. it is `rest` in `task-rest` following `task-<label>` specification
```

Comment:
me whining: I am so desperately waiting for `-` or `+` to become allowed!  in case of derivatives `_desc` it is quite hard to provide it sometimes without any way to punctuate items